falco.yaml

```diff
@@ -695,6 +695,9 @@ webserver:
   # Can be an IPV4 or IPV6 address, defaults to IPV4
   listen_address: 0.0.0.0
   k8s_healthz_endpoint: /healthz
+  # Enable the metrics endpoint providing Prometheus values
+  # It will only have an effect if metrics.enabled is set to true as well.
+  prometheus_metrics_enabled: false
```

Comment:
In addition @sgaist would you mind modifying the `metrics` config text and refer to setting this value when wanting to export Prometheus metrics?

Reply:
Sure thing !
Done